pkg/sql/ttl/ttljob/ttljob_test.go

```diff
@@ -88,12 +92,18 @@ func newRowLevelTTLTestJobTestHelper(
 
 	// As `ALTER TABLE ... SPLIT AT ...` is not supported in multi-tenancy, we
```

Comment:
this might be a problem i introduced, but this comment seems dated. maybe "we only run tests against a tenant for tests not utilising `SPLIT AT`". which your change, do multi-tenant tests work?

Reply:
It gives this error
```
    ttljob_test.go:389: error executing 'SHOW RANGES FROM TABLE tbl': pq: RangeIterator failed to seek to /Meta2/"\x00": rpc error: code = Unauthenticated desc = requested key /Meta2/"\x00" not fully contained in tenant keyspace /Tenant/1{0-1}
```